src/pubsub.js

```diff
@@ -69,6 +69,22 @@
         subscriber( message, data );
     }
 
+    function findWildCardMessage(topic)
+    {
+        var foundMessages = Object.keys(messages).find(function(messageItem){
```

Comment:
`Object.keys` was introduced in ES5.1, this project is meant to be ES3 compatible.
There are other ways of finding a specific message.